templates/pages/account/return-saved.html

```diff
@@ -7,6 +7,10 @@ <h2 class="page-heading">{{lang 'account.returns.heading' }}</h2>
         <p>
             {{lang 'account.returns.successful'}}
         </p>
+        {{#if this.instructions}}
+            <h3>Return Instructions:</h3>
```

Comment:
Can you make this a language string in https://github.com/bigcommerce/cornerstone/blob/master/lang/en.json ? This will make it easier to customize and translate; we should avoid hardcoding human-readable language in the theme HTML.

Reply:
Ok, will do.  Thanks for pointing that out.